src/Metadata/Driver/AbstractDoctrineTypeDriver.php

```diff
@@ -33,6 +33,7 @@ abstract class AbstractDoctrineTypeDriver implements DriverInterface
         'text' => 'string',
         'blob' => 'string',
         'guid' => 'string',
+        'decimal' => 'float',
```

Comment:
@goetas I'm so sorry I messed up -_-. This should be string. Do you want me to open a new PR ?